src/lib/rule-finder.js

```diff
@@ -103,6 +108,6 @@ function RuleFinder(specifiedFile, noCore) {
   this.getUnusedRules = () => getSortedRules(unusedRules);
 }
 
-module.exports = function (specifiedFile, noCore) {
-  return new RuleFinder(specifiedFile, noCore);
+module.exports = function (specifiedFile, options = {}) {
```

Comment:
Yeah, that's it, thanks.  I can't tell you how many times I've forgotten that there's that other function sitting there.